doc/api/child_process.md

```diff
@@ -281,9 +284,11 @@ changes:
 The `child_process.execFile()` function is similar to [`child_process.exec()`][]
 except that it does not spawn a shell by default. Rather, the specified
 executable `file` is spawned directly as a new process making it slightly more
-efficient than [`child_process.exec()`][].
+efficient than [`child_process.exec()`][]. Also, `child_process.execFile()` takes
```

Comment:
The efficiency is less of an issue than that shells do metacharacter expansion, so no shell will not.